client/js/analysis.js

```diff
@@ -1,15 +1,127 @@
-// Listen for on submit click
-let formSubmit = document.getElementById("formSubmit");
-formSubmit.addEventListener("click", fetchAndShowResponse, false);
-
-/**
- * Grabs response from server
- */ 
-async function fetchAndShowResponse() {
-  const response = await fetch('/api/musix');
-  const value = await response.json();
-  const el = document.getElementById(elId); 
-  el.innerText = value;
-  return value;
+const COMMENT_APPEARANCE_TIME = 1500;
+const TOTAL_COMMENTS_TO_DISPLAY = 6;
+const TOTAL_COMMENTS_TIME = COMMENT_APPEARANCE_TIME * TOTAL_COMMENTS_TO_DISPLAY;
+
+window.onload = function() {
+  for (let i = 1; i < 6; i++) {
+    setTimeout(() => {
+      addListElement(i);
+    }, i * COMMENT_APPEARANCE_TIME);
+  }
+  setTimeout(() => {
+    addFeedbackResult('POSITIVE');
+  }, TOTAL_COMMENTS_TO_DISPLAY);
+
+  for (let i = 0; i < 6; i++) {
+    const values = [
+      'TOXICITY',
+      'IDENTITY ATTACK',
+      'INSULT',
+      'PROFANITY',
+      'THREAT',
+      'FLIRTATION',
+    ];
+    // Mock the data with a number between 30% and 60% for now.
+    const chartFullness = Math.floor(Math.random() * 61 + 30);
+    addDonutChart(values[i], chartFullness);
+  }
+};
+
+/**
+ * Adds a bullet point to the comment list
+ *
+ * @param {string} str the txt of the bullet point
+ */
+function addListElement(str) {
+  const el = document.createElement('li');
+  el.innerHTML = str;
+  document.getElementById('list').appendChild(el);
+}
+
+/**
+ * Adds the feedback result element to the DOM tree
+ *
+ * @param {string} result Positive, Negative, or Neutral
+ */
+function addFeedbackResult(result) {
+  const el = document.createElement('div');
+  el.classList = 'center fade';
+  const text = document.createElement('h1');
+  text.setAttribute('id', 'overall');
+  text.innerHTML = 'Overall Response\n: ' + result;
+  el.appendChild(text);
+
+  const list = document.getElementById('list');
+  list.appendChild(el);
+}
+
+/**
+ * Adds a donut chart
+ *
+ * @param {string} str the header string of the circle
+ * @param {string} percent the amount the circle will be filled
+ */
+function addDonutChart(str, percent) {
+  const div = document.createElement('div');
+
+  div.setAttribute('id', 'a-chart');
```

Comment:
Use a class instead of an id. Id's should be unique on your page, and you create 6 of these charts.